scheduler/plugin/guest.go

```diff
@@ -67,26 +68,47 @@ func (pl *wasmPlugin) newGuest(ctx context.Context) (*guest, error) {
 		out.Reset()
 	}
 
+	// Allocate a call stack sized to max of params / return values of any
```

Comment:
Thanks, didn't realize more functions were coming so soon 😁

Because of that I didn't implement an idea of computing the size in `detectExports`, just an idea probably for another PR anyways

Reply:
yep, the route I'm trying for is (and seeing how big even one change is!) to contain features knowing they are incomplete or small debt is accumulating. Like the two we've implemented are missing a parameter, then solve for that specific parameter as a PR. In that line, once we have a different size param/return sum we can solve it then.